docs(README): update token comment (#270)
Topsort/analytics.js: README.md

```html
<script>
  window.TS = {

>     token: "<YOUR-TOPSORT.JS-TOKEN>", // This token is provided by Topsort and you'll have one for each of your environments

Reviewer: ```suggestion
    token: "<YOUR-TOPSORT.JS-TOKEN>", // Generate a token for each environment in the Topsort Auction Manager
```
